### DISCUSSION TITLE
[Bug] opa backend refuses connection from satellite

### DISCUSSION THREAD

**Discussion by leoudimg** [apache/skywalking#11150] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Satellite (apache/skywalking-satellite)

### What happened

Hi,

I made all the configurations as instructed in git hub, but when I access the UI, I have no data, searching in the logs I found many logs about connection refused as you can see in the file attached:
[satellite2.log](https://github.com/apache/skywalking/files/12208271/satellite2.log)

Bellow I put the command that I used to install the components using Helm:

Can you help me to solve this problem?

### What you expected to happen

I expect that satellite pod could send the information to the OAP servers.

### How to reproduce

Creating the components with the instructions below:

export SKYWALKING_RELEASE_VERSION=4.5.0  # change the release version according to your need
export SKYWALKING_RELEASE_NAME=skywalking  # change the release name according to your scenario
export SKYWALKING_RELEASE_NAMESPACE=skywalking  # change the namespace to where you want to install SkyWalking

helm install "${SKYWALKING_RELEASE_NAME}" \
  oci://registry-1.docker.io/apache/skywalking-helm \
  --version "${SKYWALKING_RELEASE_VERSION}" \
  -n "${SKYWALKING_RELEASE_NAMESPACE}" \
  --set oap.image.tag=9.5.0 \
  --set elasticsearch.replicas=1 \
  --set elasticsearch.minimumMasterNodes=1 \
  --set elasticsearch.volumeClaimTemplate.resources.requests.storage=10Gi \
  --set oap.storageType=elasticsearch \
  --set ui.image.tag=9.5.0 \
  --set satellite.enabled=true \
  --set satellite.image.tag=v1.2.0

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
@mrproliu Would you able to find any clue from the logs? It is not clear to me.

**Reply by mrproliu** [reply]
I think the log is normal because both OAP and Satellite start at the same time, but OAP takes longer to expose its port externally, so a brief confusion is normal. At the end of the log, it will output: "the client connection of the sender module connected."

So, after waiting for a while now, will there still be no data?
